test/sanity/validate-modules/schema.py

```diff
@@ -17,29 +17,33 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from voluptuous import ALLOW_EXTRA, Any, Required, Schema
+from ansible.compat.six import string_types
+# Because string_types from six is tuple mostly used in isinstance()
+# function and we need just class name for schema definition
+string_types = string_types[0]
```

Comment:
That may have to be `*string_types` and `list(string_types)`.

Maybe create `string_types_list = list(string_types)` so you avoid recasting to a list multiple times.